src/wallet/test/coinselector_tests.cpp

```diff
@@ -299,9 +299,21 @@ BOOST_AUTO_TEST_CASE(bnb_search_test)
         /*tx_noinputs_size=*/ 0,
         /*avoid_partial=*/ false,
     };
-    coin_selection_params_bnb.m_change_fee = coin_selection_params_bnb.m_effective_feerate.GetFee(coin_selection_params_bnb.change_output_size);
-    coin_selection_params_bnb.m_cost_of_change = coin_selection_params_bnb.m_effective_feerate.GetFee(coin_selection_params_bnb.change_spend_size) + coin_selection_params_bnb.m_change_fee;
-    coin_selection_params_bnb.min_viable_change = coin_selection_params_bnb.m_effective_feerate.GetFee(coin_selection_params_bnb.change_spend_size);
+
+    // This is the fee to create a change output which is 68 bytes.
+    // fee = (change_spend_size * effective_feerate) / 1000
+    // fee = (68 * 3000) / 1000
+    CAmount fee = coin_selection_params_bnb.m_effective_feerate.GetFee(
+                coin_selection_params_bnb.change_spend_size);
+    BOOST_CHECK_EQUAL(CAmount(204), fee);
+
+    // cost_of_change = current fee to create the output + future fee to spend the output.
+    // cost of change = (change_spend_size * effective feerate) + (change_output_size * discard_feerate)
+    // 297 = ((68 * 3000) / 1000) + (31 * 3000) / 1000
+    coin_selection_params_bnb.m_cost_of_change = fee + coin_selection_params_bnb.m_change_fee;
+    BOOST_CHECK_EQUAL(CAmount(297), coin_selection_params_bnb.min_viable_change);
```

Comment:
should be `coin_selection_params_bnb.m_cost_of_change`

Reply:
@davidgumberg thanks.  Interestingly the comment used to read:
`297 = ((68 * 3000) / 1000) + (31 * 3000) / 1000` however the value of `coin_selection_params_bnb.m_change_fee` has changed since I originally opened this PR (I'm not sure why atm).

In this commit: https://github.com/bitcoin/bitcoin/commit/4fef5344288e454460b80db0316294e1ec1ad8ad the `m_change_fee` was `93`.